core/mm/core_mmu.c

```diff
@@ -73,6 +75,8 @@ static struct memaccess_area nsec_shared[] __nex_data = {
 };
 
 #if defined(CFG_SECURE_DATA_PATH)
+static const char *tz_sdp_match = "linaro,secure-heap";
+static struct memaccess_area sec_sdp;
 #ifdef CFG_TEE_SDP_MEM_BASE
```

Comment:
Could `CFG_TEE_SDP_MEM_BASE` and `CFG_TEE_SDP_MEM_SIZE` be removed from the code and the config files since the new method is to use the embedded DT? I see no benefit in allowing both methods; the upgrade seems easy enough.

Reply:
ok, do we still need to support TEE_SDP_TEST_MEM_BASE ?

Reply:
I agree with you but some platforms to use `CFG_TEE_SDP_MEM_BASE/_SIZE` (hikey :), marvel, poplar, vexpress). We should get their feedback first.

Reply:
@omasse-linaro the test pool could still be created if SDP is enabled but not defined in the embedded DT I suppose.
Anyway let's leave that for later.